extension/Userscripts Extension/Resources/content.js

```diff
@@ -323,17 +322,19 @@ function cspFallback(e) {
     }
 }
 
-browser.runtime.sendMessage({name: "REQ_USERSCRIPTS"}, response => {
+const injection = () => browser.runtime.sendMessage({name: "REQ_USERSCRIPTS"}, response => {
```

Comment:
@ACTCD 

Function [declarations](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function) are preferable when possible, unless there is good reason for an [expression](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/function). This is purely preference.

Is there any reason for the expression?